### PR TITLE
Add Favicon.ico

### LINKED ISSUE #705: Add favicon.ico
**Is your feature request related to a problem? Please describe.**
We should have favicons, or at least one BBC one. Ask UX, I found one here: http://newsvote.bbc.co.uk/favicon.ico

**Describe the solution you'd like**
Have a favicon

### PR DESCRIPTION
**Pull request by bcmn**
Resolves #705.

Previously, Lighthouse would show a warning when run against production because `/favicon.ico` would return a 404, as seen below. With this change, this is no longer true.

screen shot 2018-10-02 at 16 44 21

**Test Description**
**N.B.:** Chrome doesn't show favicons on localhost, to the best of my knowledge. Once on TEST it should work. In the meantime, if you spin up prod locally with `npm run build` & `npm run start` & then run lighthouse, you can see as Lighthouse runs that Chrome _does_ fetch it successfully.

It should still work in other browsers when running dev or prod locally. Should appear as below:

screen shot 2018-10-04 at 13 43 50

- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: src/app/components/Document/index.jsx [hunk 1]
```diff
@@ -22,6 +22,8 @@ const Document = ({ assets, app, data, styleTags, helmet }) => {
         <meta name="robots" content="noindex,nofollow" />
         <meta name="theme-color" content={C_POSTBOX} />
         {meta}
+        <link rel="icon" href="/favicon.ico" type="image/x-icon" />
```
Comment (jtart) [inline src/app/components/Document/index.jsx]: Does this second `icon` link need to exist?

Comment (bcmn) [reply src/app/components/Document/index.jsx]: Ooops. Good point, it should be covered by the line below.

**Review by jtart — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Looks good!

**Comment by bcmn** [comment]
screen shot 2018-10-04 at 13 43 50

**Pull request merged**